TestPlaybooks/playbook-McAfee_DXL_-_Test.yml

```diff
@@ -45,7 +44,6 @@ tasks:
       script: McAfee DXL|||dxl-push-ip
```

Comment:
brand name should also be removed from the script value:
`script: '|||dxl-push-ip'`

relevant for all tasks